libs/cart/src/actions/cart.actions.ts

```diff
@@ -3,27 +3,28 @@ import { Action } from '@ngrx/store';
 import { DaffCart } from '../models/cart';
 
 export enum DaffCartActionTypes {
-    CartLoadAction = '[DaffCart] Load Action',
-    CartLoadSuccessAction = '[DaffCart] Load Success Action',
-    CartLoadFailureAction = '[DaffCart] Load Failure Action',
-    AddToCartAction = '[DaffCart] Add To Cart Action',
-    AddToCartSuccessAction = '[DaffCart] Add to Cart Success Action',
-    AddToCartFailureAction = '[DaffCart] Add to Cart Failure Action',    
-    CartResetAction = '[DaffCart] Reset Action',
-    CartResetSuccessAction = '[DaffCart] Reset Success Action',
-    CartResetFailureAction = '[DaffCart] Reset Failure Action'
+  CartLoadAction = '[DaffCart] Cart Load Action',
+  CartLoadSuccessAction = '[DaffCart] Cart Load Success Action',
+  CartLoadFailureAction = '[DaffCart] Cart Load Failure Action',
+  CartCreateAction = '[DaffCart] Cart Create Action',
+  CartCreateSuccessAction = '[DaffCart] Cart Create Success Action',
+  CartCreateFailureAction = '[DaffCart] Cart Create Failure Action',
+  AddToCartAction = '[DaffCart] Cart Add To Cart Action',
+  AddToCartSuccessAction = '[DaffCart] Cart Add to Cart Success Action',
+  AddToCartFailureAction = '[DaffCart] Cart Add to Cart Failure Action',
+  CartResetAction = '[DaffCart] Cart Reset Action',
+  CartResetSuccessAction = '[DaffCart] Cart Reset Success Action',
+  CartResetFailureAction = '[DaffCart] Cart Reset Failure Action'
 }
 
 export class DaffCartLoad implements Action {
```

Comment:
So the idea here is to get the id from state/localstorage? Then make a createCart call if there is no id?

Reply:
Yeah that was my idea.

Reply:
looks good then